### ISSUE TITLE
[bug] default value of ignoreempty is true contrary to documentation

### ISSUE THREAD

**Issue by scrool** [deadc0de6/dotdrop#281]
Dotdrop version (and git commit if run from source): v1.3.5
Using dotdrop: pypi

**Describe the bug**

https://dotdrop.readthedocs.io/en/latest/config-format/ says default value of `ignoreempty` in config entry is `false`. That's isn't true because it is rather `true`.

**Steps to Reproduce**

1. Follow quick start:
```
$ mkdir dotfiles && cd dotfiles
$ git init
$ git submodule add https://github.com/deadc0de6/dotdrop.git
$ pip3 install -r dotdrop/requirements.txt --user
$ ./dotdrop/bootstrap.sh
```
2. Run:
`$ ./dotdrop.sh install --verbose`
3. Observe value of `ignoreempty`:
```
[DEBUG][dotdrop.options.__init__] version: 1.3.5
[DEBUG][dotdrop.options.__init__] command: /home/user/dotfiles/dotdrop/dotdrop/dotdrop.py install --verbose
[DEBUG][dotdrop.options.__init__] config file: config.yaml
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] ----------start:/home/user/dotfiles/config.yaml----------
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] 
config:
  backup: true
  create: true
  dotpath: dotfiles
dotfiles:
profiles:
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] ----------end:/home/user/dotfiles/config.yaml----------
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] normalizing relative to cfg: dotfiles -> /home/user/dotfiles/dotfiles
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] settings block::
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] 	- "backup": True
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] 	- "banner": True
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] 	- "create": True
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] 	- "dotpath": /home/user/dotfiles/dotfiles
[DEBUG][dotdrop.cfg_yaml._dbg] [config.yaml] 	- "ignoreempty": True
...
```

**Expected behavior**

`- "ignoreempty": False`

**Additional information**

Config file created with the bootstrap doesn't modify defaults:
```yaml
config:
  backup: true
  create: true
  dotpath: dotfiles
dotfiles:
profiles:
```

I _guess_ this could be solved by either update of documentation or by update of defaults. I'd prefer update of default because it was confusing why added empty file isn't installed:
```
$ touch ~/.hushlogin
$ ./dotdrop.sh import --no-banner ~/.hushlogin
	-> "/home/user/.hushlogin" imported

1 file(s) imported.
$ rm ~/.hushlogin
$ ./dotdrop.sh install --no-banner                  

0 dotfile(s) installed.
$ stat ~/.hushlogin
stat: cannot stat '/home/user/.hushlogin': No such file or directory
```

Contents of the config file after the import:
```yaml
config:
  backup: true
  create: true
  dotpath: dotfiles
dotfiles:
  f_hushlogin:
    src: hushlogin
    dst: ~/.hushlogin
profiles:
  machine:
    dotfiles:
    - f_hushlogin
````

**Comment by deadc0de6** [comment]
Thanks for reporting this. I fixed the code and `ignoreempty` is per default false.

**Comment by scrool** [comment]
Works with v1.3.5-3-g825ffee. Thanks!

(I'm keeping this open, I'm not sure if you would like to close this only after the release or how your release process works)

**Comment by deadc0de6** [comment]
Awesome, it has been merged into the new version 1.3.6!.
Thanks for your help!
